public/index.html

```diff
@@ -6,7 +6,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <meta name="theme-color" content="#000000">
         <meta property="og:title" content="POA Bridge UI" />
-        <meta property="og:description" content="The POA cross-chain bridge serves as a method of transferring POA native tokens from the POA Network to the Ethereum network in a quick and cost-efficient manner. " />
+        <meta property="og:description" content="The TokenBridge serves as a method of transferring the Bancor Network tokens between the Ethereum network to Rootstock network in a quick and cost-efficient manner." />
```

Comment:
We need to have a separate parameter for this in the `.env` file

Reply:
Done